src/sentry/feedback/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import BoundedBigIntegerField, Model, region_silo_only_model, sane_repr
+from sentry.db.models.fields import UUIDField
+from sentry.db.models.fields.array import ArrayField
+
+
+@region_silo_only_model
+class Feedback(Model):
+    __relocation_scope__ = RelocationScope.Excluded
+
+    project_id = BoundedBigIntegerField(db_index=True)
+    replay_id = models.CharField(max_length=32, null=True, db_index=True)
+    url = models.CharField(max_length=1000, null=True)
+    error_ids = ArrayField()
+    trace_ids = ArrayField()
+    feedback_text = models.TextField()
+    event_id = UUIDField(unique=True)
```

Comment:
Is there a reason to store event_id here?  cc @JoshFerge

Reply:
this will be the id of the user feedback. does that make sense?

Reply:
Why not name it `id`?  Or `feedback_id`?  Event seems vague to me.

Reply:
https://develop.sentry.dev/sdk/event-payloads/types/

event_id is the name of ids on our events -- i believe even replay within ingest has `event_id` but we store it as replay_id. we could rename it to feedback_id (and map event_id it to feedback_id when storing it).

Reply:
to follow previous conventions, I'll keep it as `event_id` for now and map it to `feedback_id` later